ceph-releases/luminous/ganesha.repo

```diff
@@ -0,0 +1,5 @@
+[ganesha]
```

Comment:
but that will be shared with Redhat then, so to avoid the duplication, I thought putting the file here.

Reply:
It seems to me that RHEL would want to use RHEL repos for installing things into its container. I guess the answer here depends on what Red Hat wants to do to build these containers internally assuming they want to do so. It would make more sense, since they would have tighter control over the source.